Syntax highlighting for Elixir module names and atoms

feliperenan opened the pull request:
Closes: https://github.com/arcticicestudio/nord-visual-studio-code/issues/165

Improve Elixir support adding syntax highlight to atoms and module names using `vscode-elixir-lis` patterns.

References:
https://github.com/JakeBecker/vscode-elixir-ls
https://github.com/JakeBecker/vscode-elixir-ls/blob/master/syntaxes/elixir.json

feliperenan committed:
Add Elixir syntax highlight for module names and atoms.

Fix: https://github.com/arcticicestudio/nord-visual-studio-code/issues/165

Improve Elixir support adding syntax highlight to atoms and module
names using `vscode-elixir-lis` patterns.

References:
https://github.com/JakeBecker/vscode-elixir-ls
https://github.com/JakeBecker/vscode-elixir-ls/blob/master/syntaxes/elixir.json

arcticicestudio commented:
Hi @feliperenan :wave:, thanks for your contribution :+1:
I'll review it in the near future and get back to you.

arcticicestudio committed:
Add VSCode extension ID to supported Elixir scopes

This documents that the added scopes are only available for the specific
extension with the given ID and not supported by VSCode out-of-the-box.

GH-165

arcticicestudio reviewed (approved):
LGTM, thanks again for your contribution.
I've only added the marketplace ID of the extension to document that these scopes are not support out-of-the-box by VSCode but only for the specific extension.

---

###### Maintainer Release/Changelog Resources

<p align="center"><strong>Before</strong></p>
<p align="center"><img src="https://user-images.githubusercontent.com/7836623/71354925-b8376800-257d-11ea-8d64-f04538d066ae.png" /></p>

<p align="center"><strong>After</strong></p>
<p align="center"><img src="https://user-images.githubusercontent.com/7836623/71354924-b79ed180-257d-11ea-89ff-e194bb78a463.png" /></p>
